<?xml version="1.0"?>
<!-- SESSION ID: 1715076698 -->
<!-- SERIAL: 819312070183 -->
<launch>

  <!-- Sensor parameters -->
  <arg name="sensor_name" default="realsense1" />
  <arg name="sensor_id" default="819312070183" />

  <!-- Launch sensor -->
<include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
    <arg name="serial_no"        value="$(arg sensor_id)" />
      <arg name="camera"         value="$(arg sensor_name)" />
  </include>

</launch>
